src/app/reset-password/page.tsx

```diff
@@ -0,0 +1,26 @@
+"use client"
+
+import { useState } from 'react';
+import { useRouter } from 'next/navigation';
+import supabase from '../../api/supabase/createClient';
+
+export default function ResetPassword() {
+    const [newPassword, setPassword] = useState('');
```

Comment:
Pretty minor but I'd recommend changing this to `const [password, setPassword] = useState('');` to follow useState naming convention.